lib/utils.ts

```diff
@@ -205,10 +205,10 @@ export const getErrorMessage = (error: unknown) => {
   return String(error)
 }
 
-export const matchesSystemIniFile = (text: string) => {
-  const match = text.match(/; for 16-bit app support/gi)
-  return match !== null && match.length >= 1
-}
+// export const matchesSystemIniFile = (text: string) => {
+//   const match = text.match(/; for 16-bit app support/gi)
+//   return match !== null && match.length >= 1
+// }
```

Comment:
Esse bloco de código não pode estar comentado

```suggestion
 export const matchesSystemIniFile = (text: string) => {
   const match = text.match(/; for 16-bit app support/gi)
   return match !== null && match.length >= 1
 }
```